scripts/analyze_results_directory.py

```diff
@@ -1,5 +1,5 @@
 """Script to analyze experiments resulting from running the script
```

Comment:
This docstring is out of date. We can just say `"""Script to analyze results of experiments, stored in the results/ directory.`